qt/scientific_interfaces/Indirect/IndirectFitDataModel.cpp

```diff
@@ -116,13 +117,17 @@ std::vector<std::pair<std::string, size_t>> IndirectFitDataModel::getResolutions
   return resolutionVector;
 }
 
-void IndirectFitDataModel::setResolution(const std::string &name) {
-  setResolution(name, getNumberOfWorkspaces() - WorkspaceID{1});
+bool IndirectFitDataModel::setResolution(const std::string &name) {
+  return setResolution(name, getNumberOfWorkspaces() - WorkspaceID{1});
 }
 
-void IndirectFitDataModel::setResolution(const std::string &name, WorkspaceID workspaceID) {
+bool IndirectFitDataModel::setResolution(const std::string &name, WorkspaceID workspaceID) {
+  bool isValid = true;
```

Comment:
Initially, I found this variable name a bit confusing as I assumed it was utilized for all types of validation, but later realized that it was only used in null values validation. Perhaps naming it something that clearly conveys its association with null values, such as "hasValidValues," would be more appropriate.